lib/highlight.js

```diff
@@ -43,15 +42,21 @@ function highlightUtil(str, options = {}) {
     content += formatLine(line, Number(firstLine) + i, mark, options, wrap);
   }
 
+  let codeCaption = '';
+
+  if (caption) {
+    codeCaption = wrap ? `<figcaption>${caption}</figcaption>` : `<div>${caption}</div>`;
```

Comment:
```suggestion
    codeCaption = wrap ? `<figcaption>${caption}</figcaption>` : `<div class="caption">${caption}</div>`;
```

For better css matching.

Reply:
updated `highlight` and `prismjs`.